osbs/core.py

```diff
@@ -405,20 +404,12 @@ def stream_logs(self, build_id):
         while True:
             buildlogs_url = self._build_url("builds/%s/log/" % build_id,
                                             **kwargs)
-            try:
-                response = self._get(buildlogs_url, stream=1,
-                                     headers={'Connection': 'close'})
-            except OsbsNetworkException as ex:
-                # pycurl reports 'empty reply from server' as
-                # FOLLOWLOCATION. Handle this as though there were no
-                # lines returned.
-                if ex.status_code != pycurl.FOLLOWLOCATION:
-                    raise
-            else:
-                check_response(response)
-                for line in response.iter_lines():
-                    last_activity = time.time()
-                    yield line
+            response = self._get(buildlogs_url, stream=1,
+                                 headers={'Connection': 'close'})
```

Comment:
Fixed in https://github.com/vrutkovs/osbs-client/commit/a93fe430fce96f39e7fb7201b7dd1fb54a6d1f2c